mechanicalsoup/stateful_browser.py

```diff
@@ -251,7 +251,7 @@ def follow_link(self, link=None, *args, **kwargs):
         Before raising, if debug is activated, list available links in the
         page and launch a browser.
 
-        :return: Forwarded from :func:`open`.
+        :return: Forwarded from :func:`open_relative`.
```

Comment:
Since open_relative actually forwards from open, the old doc was also correct. Probably we can keep the old version here to avoid a double-indirection for the user.

Reply:
Since we have a lot of methods that are wrappers for each other, there's more double-indirection going on than just this instance. I'm going to keep it how it is for now for the sake of consistency, but as we iterate on this documentation again, I would be surprised if it didn't get modified.